source/engine/graphics/GpuAllocator.cpp

```diff
@@ -62,7 +64,7 @@ namespace nc::graphics
 
     void GpuAllocator::CopyBuffer(const vk::Buffer& sourceBuffer, const vk::Buffer& destinationBuffer, const vk::DeviceSize size)
     {
-        m_commands->ExecuteCommand([sourceBuffer, destinationBuffer, size](vk::CommandBuffer cmd)
+        m_device->ExecuteCommand([sourceBuffer, destinationBuffer, size](vk::CommandBuffer cmd)
```

Comment:
NICE! I love that we're moving more towards the DX11 simplicity of having the 'device' be the center of these calls